v2/devices/axolotl.yml

```diff
@@ -19,6 +19,18 @@ user_actions:
     description: "With the device powered off, hold Volume Down + Power."
     image: "phone_power_down"
     button: true
+  lineageos_reboot_system:
+    title: "Recovery - Reboot system now"
+    description: "Press on the top left arrow until you are back on the home screen.<br>Press on 'Reboot system now'."
+    button: true
+  lineageos_sideload:
+    title: "Recovery - Apply update from ADB"
+    description: "Please select 'Apply update' -> 'Apply from ADB'."
+    button: true
+  lineageos_wipe_data:
+    title: "Recovery - Wipe data/factory reset"
+    description: "Press on the top left arrow until you are back on the home screen.<br>Press on 'Factory reset' -> 'Format data/factory reset' -> 'Format data'."
```

Comment:
`<br>` are not getting rendered and `\n` are getting swallowed as well.

Reply:
Filed an issue to track this: https://github.com/ubports/installer-configs/issues/185

Meanwhile removed line breaks.